packages/main/src/MultiComboBox.js

```diff
@@ -0,0 +1,403 @@
+import Bootstrap from "@ui5/webcomponents-base/src/Bootstrap.js";
+import UI5Element from "@ui5/webcomponents-base/src/UI5Element.js";
+import ValueState from "@ui5/webcomponents-base/src/types/ValueState.js";
+import Function from "@ui5/webcomponents-base/src/types/Function.js";
+import { isShow, isDown } from "@ui5/webcomponents-base/src/events/PseudoEvents.js";
+
+import MultiComboBoxRenderer from "./build/compiled/MultiComboBoxRenderer.lit.js";
+import Input from "./Input.js";
+import Tokenizer from "./Tokenizer.js";
+import Token from "./Token.js";
+import Icon from "./Icon.js";
+import Popover from "./Popover.js";
+import List from "./List.js";
+import StandardListItem from "./StandardListItem.js";
+
+// Styles
+import styles from "./themes/MultiComboBox.css.js";
+
+// all themes should work via the convenience import (inlined now, switch to json when elements can be imported individyally)
+import "./ThemePropertiesProvider.js";
+
+/**
+ * @public
+ */
+const metadata = {
+	tag: "ui5-multi-combobox",
+	defaultSlot: "items",
+	slots: /** @lends sap.ui.webcomponents.main.MultiComboBox.prototype */ {
+		/**
+		 * Defines the <code>ui5-multi-combobox</code> items.
+		 * </br></br>
+		 * Example: </br>
+		 * &lt;ui5-multi-combobox></br>
+		 * &nbsp;&nbsp;&nbsp;&nbsp;&lt;ui5-li>Item #1&lt;/ui5-li></br>
+		 * &nbsp;&nbsp;&nbsp;&nbsp;&lt;ui5-li>Item #2&lt;/ui5-li></br>
+		 * &lt;/ui5-multi-combobox>
+		 * <br> <br>
+		 *
+		 * @type {HTMLElement[]}
+		 * @slot
+		 * @public
+		 */
+		items: {
+			type: HTMLElement,
+			multiple: true,
+			listenFor: { include: ["*"] },
+		},
+	},
+	properties: /** @lends sap.ui.webcomponents.main.MultiComboBox.prototype */ {
```

Comment:
Add @defaultvalue false for booleans, and @defaultvalue "" (or if there is existing, such as"None").